toml/src/main/kotlin/org/rust/toml/crates/local/CratesLocalIndexServiceImpl.kt

```diff
@@ -274,6 +279,33 @@ class CratesLocalIndexServiceImpl
         private const val CRATES_INDEX_VERSION: Int = 0
 
         private val LOG: Logger = logger<CratesLocalIndexServiceImpl>()
+
+        private val _TEST_INSTANCE: CratesLocalIndexService by lazy {
+                CratesLocalIndexServiceImpl().apply {
+                    updateCrates()
+                }
+            }
+
+        /**
+         * Simulates the behaviour of real service.
+         * Returns the service object in local [_TEST_INSTANCE], where it is being properly initialized on first access.
+         */
+        @TestOnly
+        fun getTestInstance(): CratesLocalIndexService = _TEST_INSTANCE
+    }
+
+    @TestOnly
+    @Volatile
```

Comment:
Looks like this `@Volatile` is redundant because the variable is accessed from only one thread